pkg/administration/events/domain.go

```diff
@@ -11,10 +11,11 @@ const (
 
 var (
 	moduleToDomain = map[*regexp.Regexp]string{
-		regexp.MustCompile(`^reprocessor|image/service`):         imageScanningDomain,
-		regexp.MustCompile(`^pkg/notifiers(/|$)|notifiers(/|$)`): integrationDomain,
-		regexp.MustCompile(`^externalbackups(/|$)`):              integrationDomain,
-		regexp.MustCompile(`^apitoken/expiration`):               authenticationDomain,
+		regexp.MustCompile(`^reprocessor|image/service`):               imageScanningDomain,
+		regexp.MustCompile(`^pkg/notifiers(/|$)|notifiers(/|$)`):       integrationDomain,
+		regexp.MustCompile(`^pkg/cloudsources(/|$)|cloudsources(/|$)`): integrationDomain,
```

Comment:
nitpick: Should we try and simplify the expression ?
```suggestion
		regexp.MustCompile(`^(pkg/)?cloudsources(/|$)`): integrationDomain,
```